README.md

```diff
@@ -107,6 +108,31 @@ yarn build
 
 ---
 
+## Release Notes
+
+### v1.0
+
+- Revamped UI
+- All SEPs are integrated into one tool
+  - SEP-24 and SEP-31 are now found in the Asset action drop-down menus
+- Ablity to download logs
+- Claimable Balances supported
+
+### [v1.1](https://github.com/stellar/stellar-demo-wallet/releases/tag/v1.1.0)
+
+- Fix for local CORS issue
+- Updated Sentry to log exceptions
+
+### v1.2
+
+- SEP-06 now supported
+- SEP-08 now supported
+- Fixed a [bug](https://github.com/stellar/stellar-demo-wallet/issues/188) when
+  overriding home domain
+- Fixed an [issue](https://github.com/stellar/stellar-demo-wallet/issues/196)
+  where balance amounts were being overwritten
+- Sending to Muxed Accounts is now supported
+
```

Comment:
💡  Releases are usually in descending order, we should have `1.2` at the top followed by `1.1` and finally `1.0`.

This is how we do it in the [monoerepo](https://github.com/stellar/go/blob/master/clients/horizonclient/CHANGELOG.md) and it's how Github organizes the releases in https://github.com/stellar/go/releases

Reply:
Good point. Updating the file with most recent release notes first.